src/test/java/uk/gov/dhsc/htbhf/steps/Hooks.java

```diff
@@ -21,15 +24,25 @@ public void initDriver() {
      * We need to always quit the WebDriver for BrowserStack tests.
      */
     @After
-    public void quitDriver() {
+    public void finaliseTest(Scenario scenario) {
         if (isBrowserStackProfile()) {
-            log.info("Calling quit on WebDriver, active profile is: {}", activeProfile);
-            webDriverWrapper.quitDriver();
-            //TODO MRS 2019-08-23: Need to add an extra after hook here to upload the results from the test to BrowserStack.
+            try {
+                String sessionId = getSessionId();
+                log.info("Uploading results for test with sessionId: {}", sessionId);
```

Comment:
This log message should probably be in BrowserstackResultUploader, as this class doesn't know that's what's going on.

Reply:
Yes, good point, will add to the next PR to avoid needed re-approvals.